docs/reference/filters.md

```diff
@@ -880,7 +880,7 @@ See also the [ratelimit docs](https://godoc.org/github.com/zalando/skipper/ratel
 ## ratelimit
 
 Per skipper instance calculated ratelimit, that allows number of
-requests to a backend. You need to run skipper
+requests to the backend group. You need to run skipper
```

Comment:
now: "...that allows number of requests to the backend group."
could be: "...that allows forwarding a number of requests to the backend group."

Reply:
thanks, change pushed